tests/bench/run_tests.py

```
@@ -0,0 +1,697 @@
+                if target_name:
+                    context_parts.append(f"target: {target_name}")
+                if context_parts:
+                    print("    " + ", ".join(context_parts))
+
+                try:
+                    # Determine which peer to use for the query (observer)
+                    if observer_name:
+                        query_peer = peers[observer_name]
+                    else:
+                        # Use the first peer from the first session
+                        first_session_data = list(sessions.values())[0]
```

Ruff (0.12.2)

Prefer `next(iter(sessions.values()))` over single element slice

Replace with `next(iter(sessions.values()))`

(RUF015)